flash/core/classification.py

```diff
@@ -45,6 +45,7 @@ def _build(
         num_classes: Optional[int] = None,
         loss_fn: Optional[Callable] = None,
         metrics: Union[torchmetrics.Metric, Mapping, Sequence, None] = None,
+        local_files_only: bool = False,
```

Comment:
I thing it shall be 
```suggestion
        local_files: bool = Optional[str],
```
so when a local path is provided it will be used otherwise try to load from internet...